pkg/tls/ca.go

```diff
@@ -118,7 +118,6 @@ func CreateRootCA(
 	// Configure the root certificate.
 	t := createTemplate(1, &key.PublicKey, validity)
 	t.Subject = pkix.Name{CommonName: name}
```

Comment:
Would it make sense to put here something generic like "Linkerd Root CA" instead of something specific?
Also that'd mean we could have the call `tls.GenerateRootCAWithDefaults()` in `options.go` not having to pass any name...

Reply:
After further offline discussion we decided to keep things as is :+1: